spec/index.bs

```diff
@@ -3813,6 +3818,11 @@ enum GPUPipelineStatisticName {
 
 When resolving pipeline statistics query, each result is written into uint64, and the number and order of the results written to GPU buffer matches the number and order of {{GPUPipelineStatisticName}} specified in {{GPUQuerySetDescriptor/pipelineStatistics}}.
 
+## Timestamp Query ## {#timestamp}
```

Comment:
Both of these sections should mention the GPUExtensionName that's required to use them.